Packs/CortexDataLake/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Cortex Data Lake",
     "description": "Palo Alto Networks Cortex Data Lake provides cloud-based, centralized log storage and aggregation for your on premise, virtual (private cloud and public cloud) firewalls, for Prisma Access, and for cloud-delivered services such as Cortex XDR",
     "support": "xsoar",
-    "currentVersion": "1.2.5",
+    "currentVersion": "1.2.6",
```

Comment:
consider a minor change and not a revision upon a change/addition in the fetch mechanism.